drivers/adc/adc_sam0.c

```diff
@@ -81,12 +80,10 @@ static int adc_sam0_acquisition_to_clocks(const struct device *dev,
 
 		return (int)ADC_ACQ_TIME_VALUE(acquisition_time) - 1;
 	case ADC_ACQ_TIME_MICROSECONDS:
-		scaled_acq = (uint64_t)ADC_ACQ_TIME_VALUE(acquisition_time) *
-			     1000000U;
+		scaled_acq = (uint64_t)ADC_ACQ_TIME_VALUE(acquisition_time) * 1000000U;
```

Comment:
The correct style is like below if you want to fix
```c
scaled_acq = (uint64_t)ADC_ACQ_TIME_VALUE(acquisition_time)
           * 1000000U;
```

Reply:
reverted